package.json

```diff
@@ -30,13 +30,22 @@
     "changelog": "./scripts/generate-changelog.sh",
     "release": "yarn run build && lerna publish",
     "version": "yarn run changelog && git add CHANGELOG.md",
-    "process-icon-svgs": "svgo -o packages/cfpb-icons/src/cfpb-icons --enable=addClassesToSVGElement --enable=removeStyleElement --enable=removeAttrs --config='{ \"plugins\": [ { \"addClassesToSVGElement\": { \"className\": \"cf-icon-svg\" } }, { \"removeAttrs\": { \"attrs\": [ \"fill\", \"path:class\", \"circle:class\" ] } }, { \"cleanupIDs\": { \"force\": \"true\" } } ] }'"
+    "process-icon-svgs": "svgo -o packages/cfpb-icons/src/cfpb-icons --enable=addClassesToSVGElement --enable=removeStyleElement --enable=removeAttrs --config='{ \"plugins\": [ { \"addClassesToSVGElement\": { \"className\": \"cf-icon-svg\" } }, { \"removeAttrs\": { \"attrs\": [ \"fill\", \"path:class\", \"circle:class\" ] } }, { \"cleanupIDs\": { \"force\": \"true\" } } ] }'",
+    "webdriver": "wdio run test/browser/webdriver.conf.js",
```

Comment:
Ideally we should separate the command from the implementation, so we might think about giving this a more general name like in cfgov-refresh, like `test-acceptance`

Reply:
How about `test:browser` for now? Until we figure out exactly what we're going to do with webdriver (unit, integration, acceptance)

Reply:
That's better than webdriver I think and seems to be what they get called in the team. It's still a bit implementation-centric though. As an analogous example, we wouldn't call the unit tests "test:cli" or something because they run on the command-line. That's my only point, if there's something that describes the type of test, not where it runs, that would be better.